parts/k8s/cloud-init/artifacts/cse_customcloud.sh

```diff
@@ -1,12 +1,12 @@
 #!/bin/bash
 
-ensureCertificates() {
+ensureAzureStackCertificates() {
```

Comment:
Let's wrap this func in the go template layer conditional `{{- if IsAzureStackCloud}}` (and then `{{end}}` at the end, so that we save cloud-init payload in non-Azure Stack scenarios.